lib/irc.js

```diff
@@ -749,13 +749,15 @@ Client.prototype.connect = function(retryCount, callback) {
         }
 
         lines.forEach(function iterator(line) {
-            var message = parseMessage(line, self.opt.stripColors);
-
-            try {
-                self.emit('raw', message);
-            } catch (err) {
-                if (!self.conn.requestedDisconnect) {
-                    throw err;
+            if (line.length > 0) {
```

Comment:
Probably best just to check the truthyness of `line`. The empty string will evaluate as false.